### PR TITLE
Update price-adjustments.md

### PR DESCRIPTION
**Pull request by Skullsneeze**
* Fixed overriding of existing discount component
* Added sortOrder parameter for flexibility
* Adjust example of JS component so gathering of custom total is dynamic
* Minor grammar adjustments and clarifications

## Purpose of this pull request

* Update the sample code for more flexibility
* Fix some grammar issues that help clarify steps

## Affected pages

- https://developer.adobe.com/commerce/php/development/components/price-adjustments/

### CODE REVIEW

**Comment by Skullsneeze** [comment]
Quick note. The examples on this page are aimed at showing the totals on the front-end, but will not add them to the order overview in the admin view. I can add an additional section for this, but am not sure if it's best to do this as part of this PR, or instead create a new PR after this one is merged?

**Comment by jeff-matthews** [comment]
Thanks for the PR @Skullsneeze!

I think it makes sense to add the additional section in the same PR since they're related.

@goivvy, since you contributed the original topic, would mind giving this contribution a review?

**Comment by goivvy** [comment]
@jeff-matthews @Skullsneeze grammar fixes seem to be OK. Was the new code tested? The old code was tested and was working ok, just to make sure the new code works as expected.

**Comment by Skullsneeze** [comment]
@goivvy, thanks for taking a look! I used (and am using) this code and haven't ran into any issues. I'm not sure if there is another standard outside of this for testing?

@jeff-matthews I unfortunately haven't been able to add the additional examples yet, but can do so at the end of this month

**Comment by Skullsneeze** [comment]
@jeff-matthews, sorry for the delay, but I've just added the sample code for showing the adjustment as a separate row for order totals in the admin UI. I've also synced my branch with the main branch of this repository.

Let me know if you feel anything is missing or should be re-worded!

File: src/pages/development/components/price-adjustments.md [hunk 1]
```diff
@@ -412,3 +424,101 @@ To display the price-adjusted total on the checkout page, add it to the `totals`
...
+            <block class="VENDOR\MODULE\Block\Adminhtml\Sales\Order\Totals" name="custom_surcharge_total">
+                <action method="setBeforeCondition">
+                    <argument name="condition" xsi:type="string">tax</argument>
+                </action>
+            </block>
+        </referenceBlock>
+    </body>
+</page>
+```
+You can adjust the value for `condition` in the `setBeforeCondition` action to change before which row the adjustment is added.
```
Comment (jeff-matthews) [inline src/pages/development/components/price-adjustments.md:448]:
```suggestion

You can adjust the value for `condition` in the `setBeforeCondition` action to change before which row the adjustment is added.
```

File: src/pages/development/components/price-adjustments.md [hunk 2]
```diff
@@ -412,3 +424,101 @@ To display the price-adjusted total on the checkout page, add it to the `totals`
...
+            }
+        }
+        
+        return $this;
+    }
+}
+
+```
+
+If all went well you should now see a new row as part of the order totals in the admin UI whenever your price adjustment is applied.
```
Comment (jeff-matthews) [inline src/pages/development/components/price-adjustments.md:524]:
```suggestion
If all went well you should now see a new row as part of the order totals in the admin UI whenever your price adjustment is applied.

```

Comment (jeff-matthews) [reply src/pages/development/components/price-adjustments.md:524]: For fix linting error

File: src/pages/development/components/price-adjustments.md (hunk 1 above)
Comment (jeff-matthews) [reply src/pages/development/components/price-adjustments.md:448]: For fix linting error

**Review by jeff-matthews — COMMENTED** [review]
Minor suggestions to fix linting errors

**Comment by Skullsneeze** [comment]
Thanks @jeff-matthews, I've applied the suggestions :)

**Comment by jeff-matthews** [comment]
Validation failures are unrelated to changes in this PR:

```
Run yarn test
src/pages/development/backward-incompatible-changes/highlights.md
   22:37-22:204  warning  Link to https://eat.magento.com/ui/phpFqn?searchKey=TWFnZW50b1xGcmFtZXdvcmtcQ29uc29sZVxDbGk6OmdldERlZmF1bHRDb21tYW5kcw== is dead  no-dead-urls  remark-lint
  75:[12](https://github.com/AdobeDocs/commerce-php/actions/runs/8306430061/job/22784429243?pr=226#step:9:13)8-75:197  warning  Link to https://developer.ups.com/oauth-developer-guide is dead                                                                   no-dead-urls  remark-lint

⚠ 2 warnings
Error: Process completed with exit code 1.
```

Errors appear to be false positives since both resolve in manual testing. I'm going to bypass the required check and merge.

**Pull request merged**